package.json

```diff
@@ -5,31 +5,15 @@
   "author": "ChainSafe Systems",
   "license": "MIT",
   "bin": {
-    "benchmark": "./bin/index.js"
+    "benchmark": "./src/bin/index.ts"
   },
-  "files": [
-    "lib/**/*.d.ts",
-    "lib/**/*.d.ts.map",
-    "lib/**/*.js",
-    "lib/**/*.js.map",
-    "lib/**/package.json",
-    "*.d.ts",
-    "*.js",
-    "bin"
-  ],
   "type": "module",
-  "main": "./lib/cjs/index.js",
-  "module": "./lib/esm/index.js",
   "exports": {
-    ".": {
-      "import": "./lib/esm/index.js",
-      "require": "./lib/cjs/index.js"
-    }
+    ".": "./src/index.ts"
   },
   "scripts": {
-    "build": "yarn build:esm && yarn build:cjs",
-    "build:esm": "tsc -p tsconfig.build.esm.json && echo '{\"type\": \"module\"}' > ./lib/esm/package.json",
-    "build:cjs": "tsc -p tsconfig.build.cjs.json && echo '{\"type\": \"commonjs\"}' > ./lib/cjs/package.json",
+    "build:npm": "deno run -A scripts/buildNpm.ts",
```

Comment:
Why are these Deno only commands?

Reply:
We are using a package called `@deno/dnt` in these scripts. This package is a deno only package and uses Deno namespaces. This package automates the process of converting typescript only package to npm+js package.

Reply:
But guessing that means developers need to have deno installed to run the repo commands I would guess.  

Not sure if that is an issue but just thinking of loud. Also more thinking big picture about this process as I'm guessing it's something you would like to see rolled out to other repos.

Reply:
In development you don't need to build the package, it's all typescript and remains typescript.

And these specific commands are more of workflow related stuff. As we have a lot of other tools also used in Github actions, which are not needed to be installed on developer machines, though I have local tools to test Github Actions locally when I make changes in it, not everyone need it.